pkg/channel/distributed/common/testing/util.go

```diff
@@ -84,3 +89,13 @@ func RetryGet(url string, pause time.Duration, retryCount int, retryAgain int) (
 	// Request failed too many times; return the error for caller to process
 	return resp, err
 }
+
+// Sets the environment variables that are necessary for common components
+func SetTestEnvironment(t *testing.T) {
+	// The system.Namespace() call panics if the SYSTEM_NAMESPACE variable isn't set, so
+	// this sets "knative-eventing" explicitly for testing purposes
+	assert.Nil(t, os.Setenv(system.NamespaceEnvKey, "knative-eventing"))
+	// The logging.ConfigMapName() has a default if it isn't present, so this just uses that
```

Comment:
I'm assuming there's nothing special about the test value of `knative-eventing` in which case I generally prefer to use something non-standard and obviously a "test" value just to be sure?   Say... `test-system-namespace` or something?

Reply:
Yep, although it apparently can't be "dummy" because of ReviewDog